include/pistache/client.h

```diff
@@ -296,7 +296,7 @@ class RequestBuilder {
     RequestBuilder& body(const std::string& val);
     RequestBuilder& body(std::string&& val);
 
-    RequestBuilder& timeout(std::chrono::milliseconds value);
+    RequestBuilder& timeout(std::chrono::milliseconds timeout);
```

Comment:
cppcheck (linter) will complain that you are shadowing one identifier with another.  Consider naming the formal parameter "timeout_" or "timeout_ms".

Reply:
@dennisjenkins75 I've fixed this problem